### PR TITLE
Expose Deposit to other account on bridge page.

### LINKED ISSUE #393: [gateway] expose depositing to other account
## Issue Type

```
[ ] bug report
[ x] feature request
```

## Current Behavior
We're not exposing two methods of native bridging:
```
depositEthTo(account)
depositERC20To(account)
```

## Expected Behavior
```
// Deposit ETH to another l2 wallet
const depositToTx = await Proxy__L1StandardBridge.depositETHTo(
  TargetAddress, // l2 target address
  depositAmount,
  1300000, // l2 gas limit
  ethers.utils.formatBytes32String(new Date().getTime().toString()), // byte data
  {value: ETHAmount}
)
```

## Steps to Reproduce

1.
2.
3.

## Suggested Fix
Expose bridging to XYZ account on L2.

Also fix the UX (native bridge at the top, other bridges to the bottom)

### LINKED ISSUE #420: [gateway] Clear Gateway from hardcoding any Rinkeby mentions
Hardcoded URLs pointing to Rinkeby (infura), Rinkeby name used, Chain id 28 hardcoded, etc.

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.
closes: https://github.com/bobanetwork/boba/issues/393

## Overview

Enable deposit to other L2 account native bridging on gateway

## Changes

Enable deposit to other L2 account native bridging on gateway
ref screenshot for details.
Screenshot 2022-10-21 at 8 48 49 PM

## Testing

Connect to l1 chain and you will see the checkbox called `Enable Deposit to other L2 Account` as soon as you select the token once L2 address added and bridge you will have fund transfer in few moment to targeted account on L2.

https://rinkeby.etherscan.io/tx/0xcf4920f6a20b0a0cb9d1bb9a551ac399c245d3fc8148167090969908512c4084
https://rinkeby.etherscan.io/tx/0xd8dbc3f15ddc7cc25ca9dbf613895f02ad7095822328ac871a988a9687cbdc79
https://rinkeby.etherscan.io/tx/0x9419f41d2176c073f2eb54e84a9ff8d8ceab630dae796737572acaf7cf48e0e1

### CODE REVIEW

File: packages/boba/gateway/src/util/masterConfig.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ if (APP_CHAIN === 'rinkeby') {
         name: "Rinkeby",
         chainId: 4,
         chainIdHex: '0x4',
-        rpcUrl: `https://rinkeby.infura.io/v3/${INFURA_ID}`,
+        rpcUrl:  "https://rpc.ankr.com/eth_rinkeby",
```
Comment (InoMurko) [inline packages/boba/gateway/src/util/masterConfig.js]: can we pull this from env var please?

**Review by InoMurko — CHANGES_REQUESTED** [review]

File: packages/boba/gateway/src/util/masterConfig.js (hunk 1 above)
Comment (InoMurko) [reply packages/boba/gateway/src/util/masterConfig.js]: also this:
```
name: "Rinkeby",
        chainId: 4,
        chainIdHex: '0x4',
```

Comment (InoMurko) [reply packages/boba/gateway/src/util/masterConfig.js]: ah it seems this masterConfig.js is a pile of mess anyway hahaha

**Review by InoMurko — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]
LGTM 🚀

File: packages/boba/gateway/src/util/masterConfig.js (hunk 1 above)
Comment (sk-enya) [reply packages/boba/gateway/src/util/masterConfig.js]: Yeah sure will do that way...
it's good idea.

Comment (sk-enya) [reply packages/boba/gateway/src/util/masterConfig.js]: @InoMurko I update this change can you please review once so I will be good to click on merge.

File: packages/boba/gateway/.env.example [hunk 2]
```diff
@@ -13,3 +13,16 @@ REACT_APP_GA4_MEASUREMENT_ID=
...
+###############################
+REACT_APP_RINKEBY_L1_NAME=Rinkeby
+REACT_APP_RINKEBY_L1_CHAIN_ID=4
+REACT_APP_RINKEBY_L1_CHAIN_ID_HEX=0x4
+REACT_APP_RINKEBY_L1_RPC_URL=https://rpc.ankr.com/eth_rinkeby
+
+REACT_APP_RINKEBY_L2_NAME=BOBA Rinkeby L2
+REACT_APP_RINKEBY_L2_CHAIN_ID=28
+REACT_APP_RINKEBY_L2_CHAIN_ID_HEX=0x1C
+REACT_APP_RINKEBY_L2_RPC_URL=https://rinkeby.boba.network
```
Comment (InoMurko) [inline packages/boba/gateway/.env.example]: this wasn't necessary as no other network is handled this way

Comment (InoMurko) [inline packages/boba/gateway/.env.example]: based on this https://github.com/bobanetwork/boba/pull/421#discussion_r1003362694

Comment (sk-enya) [reply packages/boba/gateway/.env.example]: okay got it reverting it back.

**Comment by Mrzeven** [comment]
Would love to see direct Visa/mastercard deposits into boba

**Comment by InoMurko** [comment]
can we merge this?

**Pull request merged**